DOCS/man/lua.rst

```diff
@@ -680,13 +680,13 @@ with values found in the config-file and the command-line (in that order).
 
 Example implementation::
 
-    require 'mp.options'
     local options = {
         optionA = "defaultvalueA",
         optionB = -0.5,
         optionC = true,
     }
-    read_options(options, "myscript")
+
+    require "mp.options".read_options(options, "myscript")
```

Comment:
Maybe it should be something like `local opt = require 'mp.options'` and `opt.read_options(options, "myscript")` to reflect real world usage?

Reply:
I use `require "mp.options".read_options(options)` in my real-world scripts, why do you need to save it in a variable?

Reply:
I mean you don't. I was just thinking of what mpv's builtin scripts currently do.

Reply:
Guess it's fine as is.